README.md

```diff
@@ -58,17 +58,21 @@ load_regs/fec_reg_parts.txt.
 If you are loading regs for a new year, you will need to reset the database. To do that, run:
 ```bash
 $ cf unbind-service eregs fec-eregs-db
+$ cf service-keys fec-eregs-db
```

Comment:
Service keys are new to me. Do we need them at all, and if we do, where/how are we creating them?